app/app_iface.go

```diff
@@ -80,7 +80,7 @@ type AppIface interface {
 	CreateDefaultChannels(teamID string) ([]*model.Channel, *model.AppError)
 	// CreateDefaultMemberships adds users to teams and channels based on their group memberships and how those groups
 	// are configured to sync with teams and channels for group members on or after the given timestamp.
-	CreateDefaultMemberships(since int64) error
+	CreateDefaultMemberships(since int64, includeRemovedMembers bool) error
```

Comment:
Could we please update all of the comments of these interface methods with a clear, simple explanation of what the boolean parameter does?